tests/rpt/position_parameters.cc

```diff
@@ -12,7 +12,6 @@
  * the top level of the Lethe distribution.
  *
  * ---------------------------------------------------------------------
-
  *
  * Author: Audrey Collard-Daigneault Polytechnique Montreal, 2021
  */
```

Comment:
I saw that in many of the rpt related files there are specific authors in the headers, as far as I know, we are trying to keep all the files only with the general copyright header, so it might be a good opportunity to clean some of those files if you want :)

Reply:
Should every .cc and .h file have that header?